docs/components/sidebar.vue

```diff
@@ -1,5 +1,11 @@
 <template>
   <b-collapse id="bd-docs-nav" class="bd-links" tag="nav" is-nav>
+    <b-link to="/" exact router-tag="div" active-class="active">
+      <b-link to="/" exact class="bd-toc-link" active-class="">
+        BootstrapVue
```

Comment:
Maybe just "Home" would feit better here

Reply:
I thought about that... but wasn't sure if that sounded too cheezy

Reply:
I don't think so. Makes it much more clear IMHO.

Reply:
OK... lets change it to `Home`